src/ConcreteOperations/issubset.jl

```diff
@@ -493,6 +493,11 @@ function ⊆(L::LineSegment, S::ConvexSet, witness::Bool=false)
     return _issubset_line_segment(L, S)
```

Comment:
```suggestion
    return _issubset_line_segment(L, S, witness)
```

unrelated to the scope of this PR (and hence maybe can be left for a separated one) but currently `witness` is not passed to the inner function, I think it should be this way? (this also prob means the option with `witness=true` is not tested) Also this is type-unstable in the current form